extensions/hdrhistogram/src/main/java/org/radargun/stats/HistogramOperationStats.java

```diff
@@ -33,24 +23,15 @@
    @Property(doc = "Number of significant digits. Default is 2.")
    private int digits = 2;
 
-   private AbstractHistogram histogram;
-   private SoftReference<AbstractHistogram> soft;
```

Comment:
Fyi, the soft reference stuff et all is here because we had problems when the whole histogram was moved to master - as several slaves sent the histograms for hundreds of threads the master blew up running out of memory.

Reply:
Thanks for letting me know the reason for SoftReference. I will address this later.